src/k8s/pkg/k8sd/api/capi_access_handler_test.go

```diff
@@ -0,0 +1,90 @@
+package api_test
+
+import (
+	"context"
+	"database/sql"
+	"net/http"
+	"testing"
+
+	"github.com/canonical/k8s/pkg/k8sd/api"
+	"github.com/canonical/k8s/pkg/k8sd/database"
+	microcluster_testenv "github.com/canonical/k8s/pkg/utils/microcluster"
```

Comment:
nit: not sure if we need to be that verbose but fine for me

Reply:
Changed to testenv